lib/CultureFeed/Cdb/Data/DetailList.php

```diff
@@ -79,11 +79,12 @@ public function valid()
      */
     public function getDetailByLanguage($language_code)
```

Comment:
Can't we add return types to these functions, or is that out of scope? Suprised phpstan does not require them.